Utils/pp.py

```diff
@@ -62,6 +62,18 @@ def scaleHPOD(hpod: float, enabledMods):
 	
 	return round(newOd, 1)
 
+# Scales the bpm for HT/DT.
+def scaleBPM(bpm, enabledMods):
+	bpm = int(bpm)
+
+	if enabledMods & mods['HalfTime']:
```

Comment:
idk whether you wrote this but please please change it to `if enabledMods and mods['HalfTime']:`

Reply:
No, this has to be bitwise AND. Basically, every mod in the mods dict has only one bit set. The ``enabledMods`` value may have multiple bits set, hence we check via bitwise AND.
Please check the [osu!api wiki Mods section](https://github.com/ppy/osu-api/wiki#mods) for all of the values.

Reply:
oops that's my bad, i totally thought `enabledMods` was a boolean